### DISCUSSION TITLE
Can you /trust someone only for a specific claim of yours?

### DISCUSSION THREAD

**Discussion by vijaypondini** [GriefPrevention/GriefPrevention#1490] [Q&A]
Just wondering because a user trusted someone for their farm, but got trust perms for every claim he had made for. Is there any way to give permission for a specific landclaim?

**Comment by bigpresh** [comment]
Yes - use `/trust` whilst standing in the claim you want to give them trust on.  If you're not in a claim, it will instead trust them on all your claims (but I'm pretty sure it will prompt you for that first)

**Reply by hahaluckyme** [reply]
> If you're not in a claim, it will instead trust them on all your claims (but I'm pretty sure it will prompt you for that first)

It does not prompt you, it will override all your claims immediately.
